app/src/main/res/layout/activity_main.xml

```diff
@@ -10,14 +9,14 @@
     android:screenOrientation="portrait">
 
     <TextView
-        android:id="@+id/source_language_request_text"
```

Comment:
What is the purpose? I don't quit get...

Reply:
This is usability feature. Just change text field. Because it is more reasonable that user sees target language first on his device system language, while target languages depend on source and available on corresponding language